app/src/main/res/menu/menu_action_mode.xml

```diff
@@ -22,4 +22,12 @@
         android:icon="@drawable/ic_delete_white"
         android:tint="?attr/iconColorPrimary"
         app:showAsAction="always"/>
+
+    <item
+        android:id="@+id/action_share_qr"
+        android:title="@string/action_delete"
+        android:orderInCategory="100"
+        android:icon="@drawable/ic_qrcode_scan"
```

Comment:
I think a regular QR code icon (not the scan one) would fit better here. What do you think?